api/v1alpha1/controllermanagerconfig_types.go

```diff
@@ -32,6 +32,9 @@ type ControllerManagerConfig struct {
 	// ControllerManagerConfigurationSpec returns the contfigurations for controllers
 	ControllerManagerConfigurationSpec `json:",inline"`
 
+	// AutoPopulateDNSNames indicates whether or not to auto populate service DNS names.
+	AutoPopulateDNSNames bool `json:"autoPopulateDNSNames,omitempty"`
+
```

Comment:
I think this extra controller manager configurable is redundant. The feature can be toggled on the crd.

Reply:
Removed the extra configurable.